modules/cli/pages/index.adoc

```diff
@@ -348,15 +348,29 @@ $ skupper service bind hello-world-backend deployment hello-world-backend
 
 This section shows how to expose a service from a namespace where {skupper-name} is not deployed.
 
+{skupper-name} allows you expose services from other namespaces by default. 
+You can expose other resources, for example deployments if you create a site as described in this section. 
+
 .Prerequisites
 
 * A namespace where {skupper-name} is deployed.
-* A service running in a different namespace.
 * A network policy that allows communication between the namespaces
+* cluster-admin permissions if you want to expose resources other than services
+
+
+. Create a site with cluster permissions if you want to expose a Kubernetes resource other than a service:
++
+--
+----
+$ skupper init --enable-cluster-permissions
+----
+--
 
-. Log into your cluster and change to the namespace for your site.
 
 . Expose the service on the {service-network}:
+
+
+.. If you want to expose a service, for example, if you did not create a site with `--enable-cluster-permissions`:
```

Comment:
You can expose a service with a namespace qualification without requiring the extra cluster permissions. You only need those extra permissions if you want to expose a specific workload type (deployment, statefulset ect) in another namespace. The reason being that skupper then needs to be able to track the pods for that workload individually, hence the need for extra permissions.

Reply:
trying to capture this, but also avoiding explaining how it works